torch/_torch_docs.py

```diff
@@ -9836,6 +9836,33 @@ def merge_dicts(*dicts):
 `1` appears `n2` times, `2` appears `n3` times, etc.
 """.format(**common_args))
 
+add_docstr(torch.tile,
+           r"""
+tile(input, reps) -> Tensor
+
+Constructs a tensor by repeating the elements of an :attr:`input`'s dimension 
+the number of times given by the value of :attr:`reps`'s dimension.
```

Comment:
This looks pretty good. The last thing this needs to do is explain the padding behavior when there are more reps than dimensions in input, and more dimensions in input than values in reps. The NumPy documentation's description of these cases is actually pretty clear, so maybe that can be adapted?